### PR TITLE
Cdc titer

### PR DESCRIPTION
**Pull request by barneypotter24**
Divide tdb/upload.py into separate subclasses for NIMR documents, CDC documents, and eLife documents.

### CODE REVIEW

File: tdb/parse.py [hunk 1]
```diff
@@ -13,6 +13,7 @@ class parse(object):
     def __init__(self, **kwargs):
         self.table_column_names = ['viruses', 'other', 'collection', 'passage', '']
         self.titer_values = ['10.0', '20.0', '40.0', '80.0', '160.0', '320.0', '640.0', '1280.0', '2560.0', '5120.0', 'nan']
+        self.req_fields = ['ferret_id']
```
Comment (trvrb) [inline tdb/parse.py]: Shouldn't this be `serrum_id`?

File: tdb/parse.py [hunk 2]
```diff
@@ -42,32 +43,39 @@ def read_flat(self, path, fstem, **kwargs):
...
+                #     1: 'serum_strain',
+                #     2: 'ferret_id',
+                #     3: 'source',
+                #     4: 'titer'
+                # }
                 for row in table_reader:
-                    m = {key: row[ii] if ii < len(row) else "" for ii, key in header.items()}
-                    if re.search(r'[Ee][Gg][Gg]', m['ferret_id']):  # TODO FIX THIS FOR LATER IMPORTS
+                    m = {key: row[ii] if ii < len(row) else "" for ii, key in headers.items()}
+                    if 'ferret_id' in m.keys():
```
Comment (trvrb) [inline tdb/parse.py]: @barneypotter24: Same here, shouldn't this be `serum_id`?

**Comment by trvrb** [comment]
I'm getting an error with `python tdb/download.py -db test_tdb_2 -v flu --subtype h3n2`.

```
Traceback (most recent call last):
  File "tdb/download.py", line 166, in <module>
    connTDB.download(**args.__dict__)
  File "tdb/download.py", line 67, in download
    self.output(measurements, subtype=subtype, **kwargs)
  File "tdb/download.py", line 127, in output
    raise Exception("Can't output to that file type, only json or text allowed")
Exception: Can't output to that file type, only json or text allowed
```

File: tdb/parse.py (hunk 1 above)
Comment (barneypotter24) [inline tdb/parse.py]: @trvrb I believe this line is an artifact of debugging. I will remove it.

File: tdb/parse.py (hunk 2 above)
Comment (barneypotter24) [inline tdb/parse.py]: @trvrb This one should actually be `ferret_id`, this is looking for names that were scraped from the table, and in the next lines it changed the `ferret_id` field to be `serum_id`.

**Comment by trvrb** [comment]
Nice work!

**Pull request merged**